docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,320 @@
+* **`ClientRootCAs`**: (required when using the operations service) Similar to the client root CA cert file in TLS, it contains a list of client root CA certificates that can be used to verify client certificates. If the client enrolled with the orderer organization CA, then this value is the orderer organization root CA cert.
+
+## Metrics.*
+
+By default this is disabled, but if you want to monitor the metrics for the orderer, you need to use `StatsD` as your metric provider. `StatsD` uses a "push" model, pushing metrics from the orering node to a `StatsD` endpoint. Because of this, it does not require configuration of the operations service itself. See the list of [Available metrics for the orderer](../metrics_reference.html#orderer-metrics).
+
+Because Prometheus utilizes a "pull" model there is not any configuration required, beyond making the operations service available. Rather, Prometheus will send requests to the operations URL to poll for available metrics.
+
+```
+    # The metrics provider is one of statsd, prometheus, or disabled
+    Provider: disabled
+
+    # The statsd configuration
+    Statsd:
+      # network type: tcp or udp
+      Network: udp
+
+      # the statsd server address
+      Address: 127.0.0.1:8125
+
+      # The interval at which locally cached counters and gauges are pushed
+      # to statsd; timings are pushed immediately
+      WriteInterval: 30s
+
+      # The prefix is prepended to all emitted statsd metrics
+      Prefix:
+```
+
+* **`Provider`**: Must be `enabled` to use `StatsD` or `Prometheus` metrics for the ordering node.
+* **`Statsd.Address`**: (required to use `StatsD` metrics for the ordering node) When `StatsD` is enabled, you will need to configure the `hostname` and `port` of the `StatsD` server so that the ordering node can push metric updates.
+
+## Consensus.*
+
+These values could vary depending on consensus plugin. For instance, a BFT plugin would very likely have a different set of options here.
+
+```
+# The allowed key-value pairs here depend on consensus plugin. For etcd/raft,
+# we use following options:
+
+# WALDir specifies the location at which Write Ahead Logs for etcd/raft are
+# stored. Each channel will have its own subdir named after channel ID.
+WALDir: /var/hyperledger/production/orderer/etcdraft/wal
+
+# SnapDir specifies the location at which snapshots for etcd/raft are
+# stored. Each channel will have its own subdir named after channel ID.
+SnapDir: /var/hyperledger/production/orderer/etcdraft/snapshot
+```
+
+* **`WALDir`**: (default value should be overridden.) This is the path to the write ahead logs on the local filesystem of the ordering node. It can be an absolute path or relative to `FABRIC_CFG_PATH`. It defaults to `/var/hyperledger/production/orderer/etcdraft/wal`. Each channel will have its own subdirectory named after the channel ID. The user running the ordering node needs to own and have write access to this directory.
```

Comment:
it's a little odd that for WALDir and SnapDir, we don't mention that these needs to be on volumes, but we do for the ledger.  It's critical that these not be destroyed (and we have seen a lot of users make the mistake of not preserving them).